app/controllers/posts_controller.rb

```diff
@@ -25,9 +25,19 @@ def edit
   # POST /posts
   def create
     @post = Post.new(post_params)
-
     if @post.save
-      redirect_to @post, notice: 'Post was successfully created.'
+      tag_params.split(",").each do |tag_new|
```

Comment:
Nie ma sprytnej metody, chyba ze ci chodzi o query zamiast przejscia po calej tabeli.
Chodzi o to, zeby caly kod ktory napisales podzielic na mniejsze metody, ale wciaz twoje metody.